.github/workflows/post-release.yml

```diff
@@ -28,11 +28,11 @@ jobs:
           current_version=$(cargo metadata --format-version 1 --no-deps | \
             jq --raw-output '.packages | .[] | select(.name == "bevy").version')
           # Sanity check: current version should be 0.X.Y
-          if ! grep -q '^0\.[0-9]\+\.[0-9]\+$' <<< "${current_version}"; then
-            echo "Invalid version (not in 0.X.Y format): ${current_version}"
+          if ! grep -q '^0\.[0-9]\+\.[0-9]\+-dev$' <<< "${current_version}"; then
```

Comment:
Hold up, how does this work in the first place when \+ is escaped in [0-9]\+? Am I missing something?

Reply:
I'm guessing some form of mixed between shell script escaping and regex escaping, and I'm not a fan of shell script enough to try and know for sure

Reply:
@janhohenheim, grep regex syntax is different from your favorite regex syntax

in grep, `+` is treated as an ordinary character, and `\+` as a control symbol for repetition:

 - `grep '.+'` means "any character followed by literal plus", js equivalent: `/.\+/`
 - `grep '\.\+'` means "literal dot repeated at least once", js equivalent: `/\.+/`

so if you wanna start learning shell scripting, good luck, have fun :D

Reply:
@rlidwka oh wow, TIL